client/src/reducers/controls.ts

```diff
@@ -37,13 +39,31 @@ const minimizeMaximizePopUps = (
   };
 };
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
+interface ControlsState {
+  loading: boolean;
+  error: Error | string | null;
+  resettingInterface: boolean;
+  graphInteractionMode: string;
```

Comment:
could this be just the following strings?

```suggestion
  graphInteractionMode: "zoom" | "select";
```